# Norwegian translation of pybliographer (bokmål dialect).
# Copyright (C) 2000 Free Software Foundation, Inc.
# Kjartan Maraas <kmaraas@gnome.org>, 2000.
#
msgid ""
msgstr ""
"Project-Id-Version: pybliographer 1.0.3\n"
"POT-Creation-Date: 2000-09-25 13:14+ZONE\n"
"PO-Revision-Date: 2000-08-22 00:29+02:00\n"
"Last-Translator: Kjartan Maraas <kmaraas@gnome.org>\n"
"Language-Team: Norwegian <no@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=iso-8859-1\n"
"Content-Transfer-Encoding: 8-bit\n"

#: .././Pyblio/Autoload.py:57
msgid "warning: can't import %s: %s"
msgstr "advarsel: kan ikke importere %s: %s"

#: .././Pyblio/Config.py:53
msgid "value of `%s' should be of type %s"
msgstr ""

#: .././Pyblio/Config.py:196
msgid "String"
msgstr "Streng"

#: .././Pyblio/Config.py:205
msgid "Boolean"
msgstr "Boolsk"

#: .././Pyblio/Config.py:224
msgid "Integer"
msgstr "Heltall"

#: .././Pyblio/Config.py:226
msgid "Integer under %d"
msgstr ""

#: .././Pyblio/Config.py:228
msgid "Integer over %d"
msgstr ""

#: .././Pyblio/Config.py:230
msgid "Integer between %d and %d"
msgstr ""

#: .././Pyblio/Config.py:242
msgid "Element in `%s'"
msgstr ""

#: .././Pyblio/Config.py:262
msgid "Tuple (%s)"
msgstr ""

#: .././Pyblio/Config.py:283
msgid "List (%s)"
msgstr ""

#: .././Pyblio/Config.py:308
msgid "Dictionary (%s, %s)"
msgstr ""

#: .././Pyblio/Fields.py:265
msgid "Illegal year value"
msgstr ""

#: .././Pyblio/Fields.py:269
msgid "Illegal month value"
msgstr ""

#: .././Pyblio/Fields.py:273
msgid "Illegal day value"
msgstr ""

#: .././Pyblio/Fields.py:438
msgid "in %s"
msgstr ""

#: .././Pyblio/LyX.py:39
msgid "no input pipe `%s'"
msgstr ""

#: .././Pyblio/LyX.py:45
msgid "no output pipe `%s'"
msgstr ""

#: .././Pyblio/Open.py:97
msgid "method `%s' provides no opener"
msgstr ""

# .././Pyblio/Open.py:105 :151
#: .././Pyblio/Open.py:105
msgid "File `%s' does not exist"
msgstr ""

# .././Pyblio/Open.py:116 :162
#: .././Pyblio/Open.py:116
msgid "don't know how to open `%s'"
msgstr ""

#: .././Pyblio/Open.py:143
msgid "method `%s' provides no iterator"
msgstr ""

# .././Pyblio/Format/BibTeX.py:345 :403
#: .././Pyblio/Format/BibTeX.py:345
msgid "key `%s' is malformed"
msgstr ""

# .././Pyblio/Format/BibTeX.py:366 :387
#: .././Pyblio/Format/BibTeX.py:366
msgid "%s:%d: key `%s' already defined"
msgstr ""

#: .././Pyblio/GnomeUI/Editor.py:38
msgid "Last Name, First Name"
msgstr ""

#: .././Pyblio/GnomeUI/Editor.py:39
msgid "Text"
msgstr "Tekst"

#: .././Pyblio/GnomeUI/Editor.py:42
msgid "Reference"
msgstr ""

#: .././Pyblio/GnomeUI/Editor.py:258
msgid "Day"
msgstr "Dag"

#: .././Pyblio/GnomeUI/Editor.py:267
msgid "Month"
msgstr "Måned"

#: .././Pyblio/GnomeUI/Editor.py:276
msgid "Year"
msgstr "År"

#: .././Pyblio/GnomeUI/Editor.py:301
msgid "Invalid day field in date"
msgstr ""

#: .././Pyblio/GnomeUI/Editor.py:309
msgid "Invalid month field in date"
msgstr ""

#: .././Pyblio/GnomeUI/Editor.py:317
msgid "Invalid year field in date"
msgstr ""

#: .././Pyblio/GnomeUI/Editor.py:426
msgid "Entry type"
msgstr ""

#: .././Pyblio/GnomeUI/Config.py:482 .././Pyblio/GnomeUI/Editor.py:428
msgid "Key"
msgstr "Nøkkel"

#: .././Pyblio/GnomeUI/Editor.py:471
msgid "Create Field"
msgstr ""

#: .././Pyblio/GnomeUI/Editor.py:539 .././Pyblio/GnomeUI/Fields.py:191
msgid "Mandatory"
msgstr "Nødvendig"

#: .././Pyblio/GnomeUI/Editor.py:539 .././Pyblio/GnomeUI/Fields.py:192
msgid "Optional"
msgstr "Alternativ"

#: .././Pyblio/GnomeUI/Editor.py:539
msgid "Extra"
msgstr ""

#: .././Pyblio/GnomeUI/Editor.py:601
msgid "Invalid key format"
msgstr ""

#: .././Pyblio/GnomeUI/Editor.py:609
msgid "Key `%s' already exists"
msgstr ""

#: .././Pyblio/GnomeUI/Editor.py:676
msgid "Error in native string parsing"
msgstr ""

#: .././Pyblio/GnomeUI/Editor.py:689
msgid "Edit entry"
msgstr ""

# .././Pyblio/GnomeUI/Editor.py:698 :739
#: .././Pyblio/GnomeUI/Editor.py:703
msgid "Native Editing"
msgstr ""

#: .././Pyblio/GnomeUI/Editor.py:756
msgid "Standard Editing"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:193 .././Pyblio/GnomeUI/FileSelector.py:41
msgid "File"
msgstr "Fil"

#: .././Pyblio/GnomeUI/FileSelector.py:74
msgid "Bibliography type:"
msgstr ""

#: .././Pyblio/GnomeUI/Index.py:68
msgid "Add..."
msgstr "Legg til..."

#: .././Pyblio/GnomeUI/Index.py:69
msgid "Edit..."
msgstr "Rediger..."

#: .././Pyblio/GnomeUI/Index.py:70 .././Pyblio/GnomeUI/Search.py:178
msgid "Delete"
msgstr "Slett"

#: .././Pyblio/GnomeUI/Search.py:109
msgid "Search"
msgstr "Søk"

#: .././Pyblio/GnomeUI/Fields.py:225 .././Pyblio/GnomeUI/Search.py:133
msgid "Field"
msgstr ""

#: .././Pyblio/GnomeUI/Search.py:134
msgid "Pattern"
msgstr "Mønster"

#: .././Pyblio/GnomeUI/Search.py:151
msgid "Simple Search"
msgstr "Enkelt søk"

#: .././Pyblio/GnomeUI/Search.py:155
msgid "Search command:"
msgstr "Søkekommando:"

#: .././Pyblio/GnomeUI/Search.py:161
msgid "Expert Search"
msgstr "Ekspertsøk"

#: .././Pyblio/GnomeUI/Search.py:199
msgid "Full database"
msgstr "Full database"

#: .././Pyblio/GnomeUI/Search.py:243
msgid "internal error during evaluation"
msgstr "intern feil under evaluering"

#: .././Pyblio/GnomeUI/Search.py:292
msgid ""
"while compiling %s\n"
"error: %s"
msgstr ""

#: .././Pyblio/GnomeUI/Sort.py:42
msgid "Select sort criterions"
msgstr "Velg sorteringskriterier"

#: .././Pyblio/GnomeUI/Sort.py:53
msgid "Sort direction"
msgstr "Sorteringsretning"

#: .././Pyblio/GnomeUI/Sort.py:53
msgid "Sort criterions"
msgstr "Sorteringskriterier"

#: .././Pyblio/GnomeUI/Sort.py:66
msgid "[Entry Type]"
msgstr ""

#: .././Pyblio/GnomeUI/Sort.py:67
msgid "[Key Value]"
msgstr ""

#: .././Pyblio/GnomeUI/Sort.py:92
msgid "Reorder"
msgstr "Omorganiser"

#: .././Pyblio/GnomeUI/Sort.py:93
msgid "Groups the active fields on the top of the list"
msgstr ""

#: .././Pyblio/GnomeUI/Sort.py:97
msgid "Unselect all"
msgstr "Velg bort alle"

#: .././Pyblio/GnomeUI/Sort.py:98
msgid "Removes all the sort criterions"
msgstr "Fjerner alle søkekriterier"

#: .././Pyblio/GnomeUI/Utils.py:163
msgid ""
"The following errors occured:\n"
"\n"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:52
msgid "_New"
msgstr "_Ny"

#: .././Pyblio/GnomeUI/Document.py:54
msgid "_Merge with..."
msgstr "_Flett med..."

#: .././Pyblio/GnomeUI/Document.py:58
msgid "_Previous Documents"
msgstr "_Forrige dokumenter"

#: .././Pyblio/GnomeUI/Document.py:65
msgid "About..."
msgstr "Om..."

#: .././Pyblio/GnomeUI/Document.py:66
msgid "Submit a Bug Report"
msgstr "Send inn en feilrapport"

#: .././Pyblio/GnomeUI/Document.py:78
msgid "_Add..."
msgstr "_Legg til..."

#: .././Pyblio/GnomeUI/Document.py:79
msgid "_Edit..."
msgstr "R_ediger..."

#: .././Pyblio/GnomeUI/Document.py:80
msgid "_Delete"
msgstr "S_lett"

#: .././Pyblio/GnomeUI/Document.py:83
msgid "S_ort..."
msgstr "S_orter..."

# .././Pyblio/GnomeUI/Document.py:87 :101
#: .././Pyblio/GnomeUI/Document.py:87
msgid "_Cite"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:88
msgid "_Format..."
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:92
msgid "_Fields..."
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:93
msgid "_Entries..."
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:99
msgid "_File"
msgstr "_Fil"

#: .././Pyblio/GnomeUI/Document.py:100
msgid "_Edit"
msgstr "Re_diger"

#: .././Pyblio/GnomeUI/Document.py:102
msgid "_Settings"
msgstr "_Innstillinger"

#: .././Pyblio/GnomeUI/Document.py:103
msgid "_Help"
msgstr "_Hjelp"

#: .././Pyblio/GnomeUI/Document.py:107
msgid "Open"
msgstr "Åpne"

#: .././Pyblio/GnomeUI/Document.py:108
msgid "Save"
msgstr "Lagre"

#: .././Pyblio/GnomeUI/Document.py:110
msgid "Find"
msgstr "Finn"

#: .././Pyblio/GnomeUI/Document.py:111
msgid "Cite"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:113
msgid "Close"
msgstr "Lukk"

# .././Pyblio/GnomeUI/Document.py:193 :207
#: .././Pyblio/GnomeUI/Document.py:193
msgid "Previous Documents"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:229
msgid ""
"can't open file `%s' for writing:\n"
"%s"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:250
msgid ""
"Error while parsing `%s':\n"
"%s"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:269
msgid "New database"
msgstr ""

# .././Pyblio/GnomeUI/Document.py:270 :274
#: .././Pyblio/GnomeUI/Document.py:277
msgid "[no entry]"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:278
msgid "[1 entry]"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:279
msgid "[%d entries]"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:282
msgid "[%d/1 entry]"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:283
msgid "[%d/%d entries]"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:288
msgid "[modified]"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:298
msgid ""
"The database has been modified.\n"
"Discard changes ?"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:314
msgid "Merge file"
msgstr ""

# .././Pyblio/GnomeUI/Document.py:319 :379
#: .././Pyblio/GnomeUI/Document.py:326
msgid "Open error"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:350
msgid "Merge status"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:361
msgid "Open file"
msgstr "Åpne fil"

#: .././Pyblio/GnomeUI/Document.py:410
msgid ""
"Unable to save `%s':\n"
"%s"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:418
msgid ""
"An internal error occured during saving\n"
"Try to Save As..."
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:429
msgid "Save As..."
msgstr "Lagre som..."

#: .././Pyblio/GnomeUI/Document.py:435
msgid ""
"The file `%s' already exists.\n"
"Overwrite it ?"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:442
msgid ""
"During opening:\n"
"%s"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:462
msgid "Reopen error"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:503
msgid ""
"An entry called `%s' already exists.\n"
"Rename and add it anyway ?"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:540
msgid "Really remove all the entries ?"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:560
msgid "Create new entry"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:574
msgid "Really edit %d entries ?"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:612
msgid "Remove all the %d entries ?"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:614
msgid "Remove entry `%s' ?"
msgstr ""

# .././Pyblio/GnomeUI/Document.py:661 :668
#: .././Pyblio/GnomeUI/Document.py:680
msgid ""
"Can't connect to LyX:\n"
"%s"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:734
msgid ""
"Please install bug-buddy\n"
"to use this feature"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:745
msgid "This program is copyrighted under the GNU GPL"
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:747
msgid "Gnome interface to the Pybliographer system."
msgstr ""

#: .././Pyblio/GnomeUI/Document.py:752
msgid "Pybliographer Home Page"
msgstr ""

#: .././Pyblio/GnomeUI/Format.py:37
msgid "Format entries"
msgstr ""

#: .././Pyblio/GnomeUI/Format.py:51
msgid "Bibliography style:"
msgstr ""

#: .././Pyblio/GnomeUI/Format.py:52
msgid "Select style"
msgstr ""

#: .././Pyblio/GnomeUI/Format.py:58
msgid "Output format:"
msgstr ""

#: .././Pyblio/GnomeUI/Format.py:75
msgid "Output File:"
msgstr ""

#: .././Pyblio/GnomeUI/Format.py:76
msgid "Select output file"
msgstr ""

#: .././Pyblio/GnomeUI/Config.py:45
msgid "Choose your preferences"
msgstr ""

#: .././Pyblio/GnomeUI/Config.py:93
msgid ""
"Item `%s':\n"
"\n"
"%s"
msgstr ""

#: .././Pyblio/GnomeUI/Config.py:128 .././Pyblio/GnomeUI/Fields.py:157
msgid ""
"Some changes require to restart Pybliographic\n"
"to be correctly taken into account"
msgstr ""

#: .././Pyblio/GnomeUI/Config.py:249
msgid "True"
msgstr "Sann"

#: .././Pyblio/GnomeUI/Config.py:250
msgid "False"
msgstr "Usann"

#: .././Pyblio/GnomeUI/Config.py:390
msgid "Add"
msgstr "Legg til"

#: .././Pyblio/GnomeUI/Config.py:393
msgid "Update"
msgstr "Oppdater"

#: .././Pyblio/GnomeUI/Config.py:396
msgid "Remove"
msgstr "Fjern"

#: .././Pyblio/GnomeUI/Config.py:482
msgid "Value"
msgstr "Verdi"

#: .././Pyblio/GnomeUI/Config.py:491 .././Pyblio/GnomeUI/Fields.py:93
msgid "Set"
msgstr "Sett"

#: .././Pyblio/GnomeUI/Config.py:507
msgid "Key:"
msgstr "Tast:"

#: .././Pyblio/GnomeUI/Config.py:509
msgid "Value:"
msgstr "Verdi:"

#: .././Pyblio/GnomeUI/Fields.py:45
msgid "Fields configuration"
msgstr ""

#: .././Pyblio/GnomeUI/Fields.py:59
msgid "Name"
msgstr "Navn"

#: .././Pyblio/GnomeUI/Fields.py:59
msgid "Type"
msgstr "Type"

#: .././Pyblio/GnomeUI/Fields.py:70
msgid "Name:"
msgstr "Navn:"

#: .././Pyblio/GnomeUI/Fields.py:72
msgid "Type:"
msgstr "Type:"

#: .././Pyblio/GnomeUI/Fields.py:198
msgid "Entries configuration"
msgstr ""

#: .././Pyblio/GnomeUI/Fields.py:209
msgid "Entry"
msgstr ""

#: .././Pyblio/GnomeUI/Fields.py:220
msgid "Entry Name:"
msgstr ""

#: .././Pyblio/GnomeUI/Fields.py:225
msgid "Status"
msgstr "Status"

#: .././pybliographer.py:41
msgid "This is %s, version %s"
msgstr ""

# .././pybliographer.py:42 :49
#: .././pybliographer.py:45
msgid "This is free software with ABSOLUTELY NO WARRANTY."
msgstr ""

#: .././pybliographer.py:46
msgid "For details, type `warranty'."
msgstr ""

# .././pybliographer.py:110 :158
#: .././pybliographer.py:113
msgid "%s: error: can't open file `%s'"
msgstr ""

#: .././pybliographer.py:126
msgid "For help, run %s and type `help' at the prompt"
msgstr ""

#: .././pybliographer.py:135
msgid ""
"Useful commands:\n"
"\thelp     to get some help\n"
"\tquit     to quit\n"
msgstr ""

#: .././scripts/pybcompact.py:30
msgid "usage: pybcompact <latexfile> <bibtexfiles...>"
msgstr ""

#: .././scripts/pybcompact.py:34
msgid "pybcompact: error: %s\n"
msgstr ""

#: .././scripts/pybcompact.py:97
msgid "no entry"
msgstr ""

#: .././scripts/pybcompact.py:142
msgid "can't find the following entries: %s"
msgstr ""

#: .././scripts/pybconvert.py:29
msgid "usage: pybconvert <source>..<target> <input> [output]"
msgstr ""

#: .././scripts/pybconvert.py:38
msgid "pybconvert: bad conversion format"
msgstr ""

#: .././scripts/pybformat.py:34
msgid "usage: pybformat [bibtexfiles...]\n"
msgstr ""

#: .././scripts/pybformat.py:38
msgid "pybformat: error: %s\n"
msgstr ""

#: .././scripts/pybformat.py:44
msgid "pybformat: warning: %s\n"
msgstr ""

#: .././scripts/pybformat.py:141 .././scripts/pybtex.py:84
msgid "can't find style `%s'"
msgstr ""

#: .././scripts/pybtex.py:34
msgid "usage: pybtex <latexfile> [bibtexfiles...]"
msgstr ""

#: .././scripts/pybtex.py:38
msgid "pybtex: error: %s\n"
msgstr ""

#: .././scripts/pybtex.py:59
msgid "pybtex: warning: the following keys were not resolved"
msgstr ""

#: .././scripts/pybtex.py:64
msgid "no style defined"
msgstr ""
